manifest.json

```diff
@@ -22,10 +22,15 @@
   "permissions":[
     "tabs",
     "storage",
-    "bookmarks"
+    "bookmarks",
+    "https://raw.github.com/bwinton/whimsy/gh-pages/urlbar-sayings.txt"
```

Comment:
Hmm.  I think we want to load this from https://bwinton.github.io/whimsy/urlbar-sayings.txt instead…
Also, do we really need to add this as a permission to load it?  That seems odd.